README.md

```diff
@@ -1,9 +1,10 @@
-# Rkale
+# rkale
 
 ## Install
-#### Install Rkale:  
+
+Install lantern from pypi using pip or poetry:
 ```bash
-pip install rkale
+poetry add rkale
```

Comment:
Think the installation guide should be the simplest way to get it up and running.
And don't suggest the user to install things they don't need.

Reply:
Well, we only support pyproject.toml now so pip is kind of not even supported?

Reply:
That is for the project syncing, you are still free to install it how you want.
Also you can still do syncing without a pyproject config if you specify source and destination

Reply:
Okay I thought you said you were removing it?

Reply:
No was just a thought. Other people thought it useful to be able to sync with checks without a project config, so left it in.
So maybe leave the global example in the README too?